examples/ex1.cpp

```diff
@@ -88,6 +88,13 @@ int main(int argc, char *argv[])
    }
    args.PrintOptions(cout);
 
+   // FIXME
+   // 7. Set device config parameters from the command line options and switch
+   //    to working on the device.
+   Device::Configure(device);
+   Device::Print();
+   Device::Enable();
```

Comment:
I am just starting to look at the code, but what is the purpose of `Enable`? Can't it be called inside `Configure`?

Reply:
Yes, if everything works fine, we can move it there.